typing/predef.ml

```diff
@@ -493,8 +519,9 @@ let add_or_null add_type env =
      For now, we mark the type argument as [Separability.Ind] to permit
      the most argument types, and forbid arrays from accepting [or_null]s.
      In the future, we will track separability in the jkind system. *)
+  (* CR layouts v2.8: Add baggage and more mode crossing here. *)
```

Comment:
This probably crosses all axes, subject to its parameter? Except maybe externality and nullability.